### DISCUSSION TITLE
Is there a demo app that I can install and run in Windows?

### DISCUSSION THREAD

**Discussion by Tony20221** [ElectronNET/Electron.NET#862] [Q&A]
Is there a demo app that I can install and run in Windows? Download an .msi or .exe installer, install it and run it like any Windows app?

**Comment by Tony20221** [comment]
More than a year later I got no replies! It shows there's no helpful community and shows very little confidence

**Comment by softworkz** [comment]
If you are looking closely at the issues, PRs, repository content and nuget package releases, you will see a lot of recent activity while your perception is not wrong in terms of the project has been in a kind of standby mode for some time.

A loft of effort has been made, but it seems that nobody have gone through open discussions yet.

As to your question: I think the official answer is: "No, you have to create your own", I'm afraid.
